estate/views/estate_menus.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<odoo>
+    <menuitem id="estate_menu_root" name="Real Estate">
+        <menuitem id="estate_menu_advertisements" name="Advertisements">
+            <menuitem id="estate_properties_menu_action" action="estate_property_action"/>
+        </menuitem>
+    </menuitem>
+</odoo>
```

Comment:
You have to have only 1 file for all the menus in the module, combine this in estate_menus.xml